### PR TITLE
Add a parts page

### PR DESCRIPTION
**Pull request by jareddellitt**

### CODE REVIEW

File: src/components/PartRow.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        <TableCell>{row.name}</TableCell>
+        <TableCell>{row.suppliedId}</TableCell>
+        <TableCell>{row.id}</TableCell>
+        <TableCell>
+          {row.created ? new Date(row.created).toLocaleString() : undefined}
+        </TableCell>
+      </TableRow>
+      <TableRow>
+        <TableCell
+          style={{ paddingBottom: 0, paddingTop: 0 }}
```
Comment (therockstorm) [inline src/components/PartRow.tsx:91]: Nit, you can use pb and pt, pr and pl. Same with margins.

File: src/lib/part-revisions.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { PartRevisionData, PartRevisionDataAttributes } from "@vertexvis/api-client-node";
+
+import { GetPartRevisionsRes } from "../pages/api/part-revisions";
+import { Paged } from "./paging";
+
+export type PartRevision = Pick<PartRevisionData, "id"> & PartRevisionDataAttributes;
+
+export function toPartRevisionData(res: GetPartRevisionsRes): Paged<PartRevision> {
+  console.log('#######################')
```
Comment (therockstorm) [inline src/lib/part-revisions.ts:9]: Remove

**Review by therockstorm — APPROVED** [review]

File: src/lib/part-revisions.ts (hunk 2 above)
Comment (jareddellitt) [reply src/lib/part-revisions.ts]: Removed

File: src/components/PartRow.tsx (hunk 1 above)
Comment (jareddellitt) [reply src/components/PartRow.tsx]: Updated

**Review by therockstorm — APPROVED** [review]

**Pull request merged**